tests/rules/test_fix_file.py

```diff
@@ -227,9 +227,9 @@ def test_get_new_command(mocker, monkeypatch, test):
     mocker.patch('os.path.isfile', return_value=True)
     monkeypatch.setenv('EDITOR', 'dummy_editor')
 
-    cmd = Command(script=test[0], stdout=test[4], stderr=test[5])
-    #assert (get_new_command(cmd, Settings({})) ==
-    #    'dummy_editor {} +{} && {}'.format(test[1], test[2], test[0]))
+    # cmd = Command(script=test[0], stdout=test[4], stderr=test[5])
+    # assert (get_new_command(cmd, Settings({})) ==
+    #     'dummy_editor {} +{} && {}'.format(test[1], test[2], test[0]))
```

Comment:
I think we can remove this test entirely. If we ever need it, Git will give it back to us like magic.